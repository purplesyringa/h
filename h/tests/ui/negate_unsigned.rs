fn main() {
    let meow = h::map!(
        for u8;
        -1 => (),
    );
}
