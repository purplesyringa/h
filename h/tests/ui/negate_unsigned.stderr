error: cannot apply unary operator `-` to type `u8`
 --> tests/ui/negate_unsigned.rs:4:9
  |
4 |         -1 => (),
  |         ^^
